docs/TypeScript.md

```diff
@@ -126,6 +126,14 @@ The type system heavily relies on generic properties to provide the most accurat
 
 ### JSON Schema
 
+To validate your requests and responses you can use JSON Schema files. If you didn't know already, defining schemas for your Fastify routes can increase their throughput! Check out the [Validation and Serialization](Validation-and-Serialization.md) documentation for more info.
```

Comment:
This is not correct. Schema for Serialization increases throughput.

Reply:
I copied this paragraph